azurelinuxagent/common/errorstate.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime, timedelta
+
+ERROR_STATE_DELTA = timedelta(minutes=15)
```

Comment:
Is this used?

Reply:
Should also be in constants, probably

Reply:
Where?  There are quite a few constants, but I do not see one particular file dedicated to just constants.